javalib/src/main/scala/java/util/zip/CRC32.scala

```diff
@@ -25,8 +25,9 @@ class CRC32 extends Checksum {
     update(buf, 0, buf.length)
 
   def update(buf: Array[Byte], off: Int, nbytes: Int): Unit = {
-    // avoid int overflow, check null buf
-    if (off <= buf.length && nbytes >= 0 && off >= 0 && buf.length - off >= nbytes) {
+    if (nbytes == 0) {
+      ()
+    } else if (off <= buf.length && nbytes >= 0 && off >= 0 && buf.length - off >= nbytes) { // avoid int overflow, check null buf
```

Comment:
Minor nit - if you specifically check for `nbytes == 0` then should the `else if` say `&& nbytes > 0`? Also, is the comment still correct? Seems like this line would wrap with scalafmt especially with the trailing comment.

Reply:
> Also, is the comment still correct?

Not sure, not entirely sure what it was referring to.

> Seems like this line would wrap with scalafmt especially with the trailing comment.

Scalafmt in CI seems happy with the current formatting.